feature/isis/ate_tests/base_adjacencies_test/README.md

```diff
@@ -0,0 +1,153 @@
+# RT-2.1: Base IS-IS Process and Adjacencies
```

Comment:
Let's move this under `feature/experimental/isis/ate_tests/base_adjacencies_test/...` as an additional precaution so that new code will not accidentally use the deprecated packages. We will follow up with a PR to move it out of experimental once Dan's telemetry check helper is ready.